exporter/awsemfexporter/emf_exporter.go

```diff
@@ -133,14 +141,20 @@ func (emf *emfExporter) pushMetricsData(_ context.Context, md pdata.Metrics) (dr
 				err = wrapErrorIfBadRequest(&returnError)
 				return
 			}
-			returnError = pusher.ForceFlush()
```

Comment:
Is this the main performance improvement, not flushing on every metric but once per batch?

Reply:
Yes it is. And now it's in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/2572.